src/utiltest.cpp

```diff
@@ -46,6 +46,10 @@ CWalletTx GetValidReceive(ZCJoinSplit& params,
                           inputs, outputs, 2*value, 0, false};
     mtx.vjoinsplit.push_back(jsdesc);
 
+    // Shielded Output
+    OutputDescription od;
+    mtx.vShieldedOutput.push_back(od);
```

Comment:
This adds a shielded output but the transaction version is 2. It won't be valid unless the caller changes the version to 4 (and it won't allow testing pre-Sapling txns when that is intended).

Reply:
I am going to push an additional a commit adding a parameter for the version.